StreamChatClient_v3/Controllers/ChannelListControllerPublishers.swift

```diff
@@ -0,0 +1,121 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Combine
```

Comment:
This should be isolated as well.

Reply:
Interesting, it compiles without errors 😄

Reply:
You can import whatever you want, you get errors only when you actually use it.

Reply:
I'm confused with checking for existence of a framework overall 🤦‍♂️  Like `canImport`, off it always will be true 😄

Reply:
But how? I still confused. this definitely isn’t possible for 3rd party  libraries